### DISCUSSION TITLE
Does EasyTier support Virtual Lan?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [EasyTier/EasyTier#146] [Q&A]
sorry for the question, I wanted to set up a virtual LAN so I could play online with my friend

**Comment by KKRainbow** [comment]
do you mean layer2 vpn?

**Reply by Heus-Sueh** [reply]
yes

**Comment by purifierprobe** [comment]
Although Easytier is Layer3 VPN, it supports local area network games, such as Counter Strike 1.6, Warcraft III, StarCraft, and so on.

**Comment by BigSmart114514** [comment]
If you run easytier on both machines, it should work like virtual lan.
